fastestimator/op/numpyop/numpyop.py

```diff
@@ -185,7 +184,7 @@ def __init__(self,
     def forward(self, data: Union[np.ndarray, List[np.ndarray]], state: Dict[str, Any]) -> None:
         pass
 
-    def forward_batch(self, data: Union[Tensor, List[Tensor]], state: Dict[str, Any]) -> None:
+    def forward_batch(self, data: Union[torch.Tensor, List[torch.Tensor]], state: Dict[str, Any]) -> None:
```

Comment:
We should keep `Tensor = TypeVar('Tensor', torch.Tensor, np.ndarray)`. and in the type use Union[Tensor, List[Tensor]]. Because in NumpyOp, it can stipp have np array